examples/calculator/calculator.cc

```diff
@@ -158,7 +152,7 @@ struct CalculatorGrammar
 	 */
 	static const CalculatorGrammar& get()
 	{
-		static CalculatorGrammar g;
+		static CalculatorGrammar& g = *new CalculatorGrammar();
```

Comment:
I don't really like this idiom.  I can see it's useful where we care about performance more than we care about memory leaks, but it's probably bad for an example to contain code that teaches people to leak memory.